packages/flex-dev-utils/src/__tests__/fs.test.ts

```diff
@@ -227,12 +232,14 @@ describe('fs', () => {
   });
 
   describe('checkPluginConfigurationExists', () => {
-    const name = 'plugin-test';
+    const pluginJsonPath = 'test-dir-plugins';
+    const name = pluginName;
     const dir = 'test-dir';
+    const anotherDir = 'another-dir';
     const cliPath = {
-      dir: 'test-dir',
+      dir,
       flexDir: 'test-dir-flex',
-      pluginsJsonPath: 'test-dir-plugins',
+      pluginsJsonPath: pluginJsonPath,
```

Comment:
should this just be `pluginsJsonPath`

Reply:
ehh it's just a variable. I'll change it in the next PR otherwise I have to wait another 20min for PR validation to complete.